infrastructure/kctf/scripts/cluster/start.sh

```diff
@@ -18,6 +18,35 @@ gcloud compute routers nats create kctf-${CLUSTER_NAME}-nat-config --router-regi
 
 get_cluster_creds
 
+# GCSFUSE
+
+SUFFIX=$(echo "${PROJECT}-${CLUSTER_NAME}-${ZONE}" | sha1sum)
+BUCKET_NAME="kctf-gcsfuse-${SUFFIX:0:16}"
+GSA_NAME="${BUCKET_NAME}"
+GSA_EMAIL=$(gcloud iam service-accounts list --filter "name:${GSA_NAME}" --format 'get(email)' || true)
+if [ -z "${GSA_EMAIL}" ]; then
+  gcloud iam service-accounts create "${GSA_NAME}" --description "kCTF GCSFUSE service account ${CLUSTER_NAME} ${ZONE}" --display-name "kCTF GCSFUSE ${CLUSTER_NAME} ${ZONE}"
+  GSA_EMAIL=$(gcloud iam service-accounts list --filter "name:${GSA_NAME}" --format 'get(email)')
+fi
+if ! gsutil du "gs://${BUCKET_NAME}/"; then
+  gsutil mb -l eu "gs://${BUCKET_NAME}/"
+  gsutil uniformbucketlevelaccess set on "gs://${BUCKET_NAME}/"
+fi
+gcloud projects add-iam-policy-binding "${PROJECT}" --member "serviceAccount:${GSA_EMAIL}" --role roles/storage.objectAdmin
+
+KEY_PATH=$(mktemp -d)/key.json
+
+gcloud iam service-accounts keys create "${KEY_PATH}" --iam-account "${GSA_EMAIL}"
+
+kubectl create secret generic gcsfuse-secrets --from-file="${KEY_PATH}" --namespace kube-system
```

Comment:
btw, the proper way to do this would be workload identities: https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity

Reply:
I saw that, yea, unfortunately it seemed like one of the things it depends on is in gcloud alpha, which seems fragile. we might be able to move to that once its in at least beta or stable